lib/rubocop/cop/style/hash_each_methods.rb

```diff
@@ -17,10 +17,16 @@ module Style
       # @example
       #   # bad
       #   hash.keys.each { |k| p k }
-      #   hash.values.each { |v| p v }
+      #   hash.each { |k, unused_value| p k }
       #
       #   # good
       #   hash.each_key { |k| p k }
+      #
+      #   # bad
+      #   hash.values.each { |v| p v }
+      #   hash.each.each { |unused_key, v| p v }
```

Comment:
I think it should be just `hash.each` here.

Reply:
Oops! I've fixed it.